packages/caver-klay/caver-klay-kct/kctHelper.js

```diff
@@ -0,0 +1,286 @@
+/*
+    Copyright 2020 The caver-js Authors
+    This file is part of the caver-js library.
+
+    The caver-js library is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Lesser General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    The caver-js library is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU Lesser General Public License for more details.
+
+    You should have received a copy of the GNU Lesser General Public License
+    along with the caver-js. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+const _ = require('lodash')
+const BigNumber = require('bignumber.js')
+const { isBigNumber } = require('../../caver-utils')
+
+async function determineSendParams(executableObj, sendParam, defaultFrom) {
+    let { from, gas } = sendParam
+    from = from || defaultFrom
+    if (!from) throw new Error(`'from' is missing. Pass the object that from field is defined in the last parameter.`)
+
+    if (gas === undefined) {
+        const estimated = await executableObj.estimateGas({ from })
+        const originalGas = new BigNumber(estimated, 10)
+        const bufferGas = new BigNumber(1.2, 10)
+
+        gas = Math.round(originalGas.times(bufferGas))
+    }
+
+    return { from, gas, gasPrice: sendParam.gasPrice, value: sendParam.value }
+}
+
+function formatParamForUint256(param) {
+    return convertToNumberString(param)
+}
+
+function convertToNumberString(value) {
+    let numberString
+    if (_.isString(value)) value = new BigNumber(value)
```

Comment:
I think the below is also valid:
`new BigNumber(10)`

What about passing the big number directly? The code could be much simpler, then.

Reply:
Oh! okay :) I modified here logic!